pkg/messaging/v1/invoke_method_response.go

```diff
@@ -39,10 +46,13 @@ func NewInvokeMethodResponse(statusCode int32, statusMessage string, statusDetai
 }
 
 // InternalInvokeResponse returns InvokeMethodResponse for InternalInvokeResponse pb to use the helpers.
-func InternalInvokeResponse(resp *internalv1pb.InternalInvokeResponse) (*InvokeMethodResponse, error) {
-	rsp := &InvokeMethodResponse{r: resp}
-	if resp.Message == nil {
-		resp.Message = &commonv1pb.InvokeResponse{Data: &anypb.Any{Value: []byte{}}}
+func InternalInvokeResponse(pb *internalv1pb.InternalInvokeResponse) (*InvokeMethodResponse, error) {
+	rsp := &InvokeMethodResponse{r: pb}
+	if pb.Message == nil {
+		pb.Message = &commonv1pb.InvokeResponse{Data: nil}
```

Comment:
will setting nil here cause issues or is `HasMessageData` method always called before accessing Data to avoid any nil dereference?
Should that step be documented somewhere for future use?

Reply:
All methods currently access data using `RawData` or `RawDataFull` or `ProtoWithData`.